db/db_sst_test.cc

```diff
@@ -702,8 +702,9 @@ TEST_F(DBSSTTest, GetTotalSstFilesSize) {
                                        &total_sst_files_size));
   // Live SST files = 1 (compacted file)
   // Total SST files = 6 (5 original files + compacted file)
-  ASSERT_EQ(live_sst_files_size, 1 * single_file_size);
-  ASSERT_EQ(total_sst_files_size, 6 * single_file_size);
+  ASSERT_GT(live_sst_files_size, 1 * single_file_size * 0.9);
+  ASSERT_LT(live_sst_files_size, 1 * single_file_size * 1.1);
+  ASSERT_EQ(total_sst_files_size, 5 * single_file_size + live_sst_files_size);
```

Comment:
I am still trying to understand why the test doesn't work without this change ... but I can't see any issue in the rest of the code in this PR. Do you know the reason by any chance?